server/test/async-job-component.xml

```diff
@@ -108,11 +108,7 @@
     </adapters>
     <adapters key="com.cloud.agent.manager.allocator.StorageAllocator">
       <adapter name="Storage"
-        class="com.cloud.agent.manager.allocator.impl.FirstFitStorageAllocator">
```

Comment:
@bhaisaab , probably note (I would say that no one uses it) that class does not exist.